### PR TITLE
Complete review and refactoring of the Portuguese (Brazil) translation.

### PR DESCRIPTION
**Pull request by trinaldi**
Thanks to Ricardo Pereira da Costa for the carefully review.

### CODE REVIEW

**Comment by pbatard** [comment]
Thanks Tiago! For various reasons, it'll probably be a couple more weeks before I can apply this update, but I'm not forgetting your request.

**Comment by trinaldi** [comment]
No problem. Thanks, Pete.

File: res/localization/rufus.loc [hunk 1]
```diff
@@ -12383,28 +12383,36 @@ b "en-US"
 # Main dialog
 g IDD_DIALOG
 t IDS_DEVICE_TXT "Dispositivo"
-t IDS_PARTITION_TYPE_TXT "Tipo de partição e do sistema de destino"
+t IDS_PARTITION_TYPE_TXT Esquema de partição e tipo do sistema de destino"
```
Comment (luanorlandi) [inline res/localization/rufus.loc]: Missing opening quote here

**Comment by pbatard** [comment]
Can I please ask you to create a __single__ patch, on top of the current Rufus tree, and submit that as a pull request. A pull request should only ever be of a single patch, and if they all relate to updating a translation, they should be grouped. Thanks!

**Comment by trinaldi** [comment]
I'll close this PR.

**Pull request closed without merging**